docs/source/lesson/prg/python_intro.rst

```diff
@@ -1380,13 +1324,138 @@ be the package called cloudmesh_client which you can install with::
 
 If all goes well the package will be installed.
 
-Links
-=================================================================
+Alternative Installations
+-----------------------------------------------
+
+The best installation of python is provided by python.og. However
```

Comment:
The word "best" is too subjective. For this class we recommend only installing vanilla python from python.org.